lib/ansible/plugins/connection/paramiko_ssh.py

```diff
@@ -250,6 +251,24 @@
 SETTINGS_REGEX = re.compile(r'(\w+)(?:\s*=\s*|\s+)(.+)')
 
 
+def _deprecate_arg0(func):
```

Comment:
We'd need to turn new_stdin and connection into a kwargs instead, whereas I was deprecating passing it at all. This was just for 3rd party plugins potentially using it, in an abundance of caution. I'll just delete arg 0 if this seems like overkill - paramiko isn't using this now and since this a helper class to paramiko, if anything else is using this, it should have not imo.

Reply:
I've removed the arg altogether for simplicity.